docs/src/affine.md

```diff
@@ -33,23 +37,23 @@ Then given a (multivariate) standard normal ``z``, the covariance matrix of ``σ
 𝕍[σ z + μ] = Σ
 ```
 
-Comparing to the one dimensional case where
+The one-dimensional case where we have
 
 ```math
 𝕍[σ z + μ] = σ²
 ```
 
-shows that the lower Cholesky factor of the covariance generalizes the concept of standard deviation, justifying the notation.
+shows that the lower Cholesky factor of the covariance generalizes the concept of standard deviation, completing the link between ``σ`` and `Σ`.
 
 ## The "Cholesky precision" parameterization
 
-The ``(μ,σ)`` parameterization is especially convenient for random sampling. Any `z ~ Normal()` determines an `x ~ Normal(μ,σ)` through
+The ``(μ,σ)`` parameterization is especially convenient for random sampling. Any measure `z ~ Normal()` determines an `x ~ Normal(μ,σ)` through the affine transformation
 
 ```math
 x = σ z + μ
 ```
 
-On the other hand, the log-density computation is not quite so simple. Starting with an ``x``, we need to find ``z`` using
+The log-density transformation of a `Normal` with parameters μ, σ does not follow as directly. Starting with an ``x``, we need to find ``z`` using
 
 ```math
 z = σ⁻¹ (x - μ)
```

Comment:
I think there is a problem with the exponent in the maths formula, why not use $\sigma^{-1}$?

Reply:
One benefit of having it as unicode when possible is that it can render in the REPL. But we should check that the docs also render well (do they not?)

Reply:
I think they're ok